jest.config.js

```diff
@@ -38,7 +38,12 @@ module.exports = {
   },
   reporters: [
     'default',
-    ['jest-junit', { outputDirectory: '<rootDir>/tmp/junit' }],
+    ['jest-junit', {
+      outputDirectory: '<rootDir>/tmp/junit',
+      classNameTemplate: '{classname}',
+      titleTemplate: '{title}',
+      addFileAttribute: 'true',
```

Comment:
What are these for?

Reply:
They make the Junit reports on Gitlab a little clearer to read. I wrote about what each one does in the issue description here https://github.com/MatrixAI/TypeScript-Demo-Lib/issues/58 but essentially they group the tests by top-level describe and include the path to the test.

Reply:
Great that's useful.